modules/nodes-pods-vertical-autoscaler-custom-resource.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-vertical-autoscaler.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nodes-pods-vertical-autoscaler-custom-resource_{context}"]
+= Using the Vertical Pod Autoscaler Operator with Custom Resources
```

Comment:
Since this isn't really a procedure, I think it would be good to rename the title. Maybe something like "Example custom resources for the Vertical Pod Autoscaler"? WDYT?